src/node/utils/LibreOffice.js

```diff
@@ -35,7 +35,18 @@ var queue = async.queue(doConvertTask, 1);
  * @param  {Function}   callback    Standard callback function
  */
 exports.convertFile = function(srcFile, destFile, type, callback) {
-  queue.push({"srcFile": srcFile, "destFile": destFile, "type": type, "callback": callback});
+  if (type === 'doc') {
```

Comment:
as this is a very specific case, it would be good to have some comments explaining the reason we have this `if/else`. Would you mind adding it here, @ldidry ? You can add a reference to this PR, it would make it easier for other devs to know why this code needed to be added.